Deemixrr/Views/Shared/_Layout.cshtml

```diff
@@ -64,9 +64,15 @@
                 </div>
                 <ul class="navbar-nav flex-fill w-100 mb-2">
                     <li class="nav-item">
-                        <a asp-controller="Home" asp-action="Index" class="nav-link">
+                        <a class="nav-link" href="/">
                             <i class="fe fe-home fe-16"></i>
-                            <span class="ml-3 item-text">Dashboard</span>
+                            <span class="ml-3 item-text">Deemixrr Home</span>
```

Comment:
Change to `Home` Instead of `Deemixrr Home`. I guess ppl know that they are using Deemirr :)